src/main/java/com/develhope/spring/users/services/UserService.java

```diff
@@ -20,10 +26,28 @@ public class UserService {
     public User getBy(Long id) {
         return userRepository.findById(id).orElseThrow();
     }
+    void checkUserAuthorizationBy(long userId) {
```

Comment:
questo metodo deve essere privato, ma se si mergia l'autorizzazione neanche servirà più

Reply:
è package private, l'avevo messo per poter testare il metodo. Tuttavia, se questo andrà rimosso, tanto vale che lo cancello direttamente, in questo mood rimuovo completamente i problemi successivi